Tools/scripts/generate_global_objects.py

```diff
@@ -284,6 +284,12 @@ def generate_runtime_init(identifiers, strings):
                 printer.write('')
                 with printer.block('.tuple_empty =', ','):
                     printer.write('.ob_base = _PyVarObject_IMMORTAL_INIT(&PyTuple_Type, 0)')
+        printer.write('')
+        with printer.block("static inline void\n_Py_StaticStrings_Intern(void)"):
```

Comment:
Consider splitting this line for readability:

```suggestion
        printer.write("static inline void")
        with printer.block("_Py_StaticStrings_Intern(void)"):
```

or don't split the line:

```suggestion
        with printer.block("static inline void _Py_StaticStrings_Intern(void)"):
```

Reply:
done